app/Traits/ExecuteRemoteCommand.php

```
@@ -202,13 +202,13 @@ private function executeCommandWithProcess($command, $hidden, $customType, $appe
 
             if ($this->save) {
                 if (data_get($this->saved_outputs, $this->save, null) === null) {
-                    data_set($this->saved_outputs, $this->save, str());
+                    $this->saved_outputs->put($this->save, str());
                 }
                 if ($append) {
-                    $this->saved_outputs[$this->save] .= str($sanitized_output)->trim();
-                    $this->saved_outputs[$this->save] = str($this->saved_outputs[$this->save]);
+                    $current_value = $this->saved_outputs->get($this->save);
```

PHPMD (2.15.0)

The variable $current_value is not named in camelCase. (undefined)

(CamelCaseVariableName)